FusionIIIT/applications/health_center/models.py

```diff
@@ -158,7 +158,7 @@ class Ambulance_request(models.Model):
     date_request = models.DateTimeField()
     start_date = models.DateField()
     end_date = models.DateField(null=True, blank=True)
-    reason = models.CharField(max_length=50)
+    reason = models.CharField(max_length=500)
```

Comment:
The max_length for a CharField is 255. If more words are required use TextField.

Reply:
@PranshuNayak Fixed the model. PTAL.